widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -477,8 +478,13 @@ export function ConfirmWalletsModal(props: PropTypes) {
                           )}
                           value={customDestination || ''}
                           suffix={
-                            !isFirefox &&
-                            (!isMobile || !customDestination) && (
+                            customDestination ? (
+                              <IconButton
+                                onClick={handleClearCustomDestination}
+                                variant="ghost">
+                                <ClearIcon size={16} />
+                              </IconButton>
+                            ) : !!isFirefox ? null : (
```

Comment:
Reading nested ternary can be challenging

Reply:
Tnx, Fixed.